### ISSUE TITLE
[1.20.1] Moving the Rocket Controller with another contraption doesn't change its initial pos

### ISSUE THREAD

**Issue by C1ffisme** [RealAntEngineer/creating_space#68]
I built a cart for my rocket to move in and out of a KSP-style rocket building lab. I can stick the rocket to the cart when it's time to move it and unstick it when it's time to blast off.

![2024-10-21_15 10 44](https://github.com/user-attachments/assets/8e7553df-a69e-41a7-8b6c-35965771f575)

When testing this rocket for the first time, however, I noticed when I landed back on Earth it put me on the ceiling of the lab. It seems like when the rocket is moved as a separate contraption, it doesn't move the initial position of the rocket controller to compensate.

**Comment by RealAntEngineer** [comment]
Yes, it's supposed to do that. Otherwise it can create issues, but I will put something to change the landing pos later. I will come back to this post when I do it
